catalyst-gateway/bin/src/event_db/follower.rs

```diff
@@ -15,99 +16,167 @@ pub type BlockHash = String;
 /// Unique follower id
 pub type MachineId = String;
 
+/// `slot_no` column name
+const SLOT_NO_COLUMN: &str = "slot_no";
+/// `block_hash` column name
+const BLOCK_HASH_COLUMN: &str = "block_hash";
+/// `block_time` column name
+const BLOCK_TIME_COLUMN: &str = "block_time";
+/// `ended` column name
+const ENDED_COLUMN: &str = "ended";
+
+/// `insert_slot_index.sql`
+const INSERT_SLOT_INDEX_SQL: &str =
+    include_str!("../../../event-db/queries/follower/insert_slot_index.sql");
+/// `select_slot_info_by_datetime.sql.tmpl`
+const SLOT_INFO_SQL_TMPL: &str =
+    include_str!("../../../event-db/queries/follower/select_slot_info_by_datetime.sql.tmpl");
```

Comment:
Can we move both these templates to be colocated with the code.
They have no logical relationship to `../../../event-db/queries/follower` and it makes them look like they have some purpose other than being templates for this code.

Same comment for `insert_slot_index.sql`  
I would rather it lives with its code, not some unrelated other place.

Reply:
Also, lets use `.hbs` as the extension for a handlebars template, so the extension makes it clearer what kind of template language we are using.